### PR TITLE
Added a Menu management cookbook

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Hello,

we have no information in docs about how to create and customize the back office menu in Akeneo.

I'm not sure about the right place where to put this information, let's complete the "How to customize the UI" cookbook for now :)

Mickaël

### CODE REVIEW

File: cookbook/ui_customization/how_to_customize_menu.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                label: 'Manufacturers'
+                route: 'app_manufacturers_index'
+
+The example above defines three menu items:
+ - The ``reference_data`` item consists of a label and the URI ``#``. This means that the item will not react on mouse click,
+   but can be used as a placeholder for nested menus;
+ - Both ``brand`` and ``manufacturer`` items reference and existing route. So, when we will click one of theses items, we will
+   get the corresponding page.
+
+There is the list of all available options for an item:
```
Comment (MarieMinasyan) [inline cookbook/ui_customization/how_to_customize_menu.rst]: _Here is the list_

File: cookbook/ui_customization/how_to_customize_menu.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            labelAttributes: <attr_list>      # <span> attributes for text items without link
+            childrenAttributes: <attr_list>   # <ul> item attributes for nested lists
+            showNonAuthorized: <boolean>      # show for non-authorized users
+            display: <boolean>                # disable showing of menu item
+            displayChildren: <boolean>        # disable showing of menu item children
+
+Organize the Navigation trees
+*****************************
+
+The next step is to compose a tree of the menu items. All trees are build under the ``tree`` key.
```
Comment (MarieMinasyan) [inline cookbook/ui_customization/how_to_customize_menu.rst]: _to compose_ means _faire partie de_, you should say _The next step is to put together a tree for these menu items_

Comment (MarieMinasyan) [inline cookbook/ui_customization/how_to_customize_menu.rst]: _All trees are built_

**Comment by mickaelandrieu** [comment]
@MarieMinasyan fixed, thanks for review

File: cookbook/ui_customization/how_to_customize_menu.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                label: 'Brands'
+                route: 'app_brands_index'
+            manufacturer:
+                label: 'Manufacturers'
+                route: 'app_manufacturers_index'
+
+The example above defines three menu items:
+ - The ``reference_data`` item consists of a label and the URI ``#``. This means that the item will not react on mouse click,
+   but can be used as a placeholder for nested menus;
+ - Both ``brand`` and ``manufacturer`` items reference and existing route. So, when we will click one of theses items, we will
```
Comment (damien-carcel) [inline cookbook/ui_customization/how_to_customize_menu.rst]: "reference **an** existing" ?

File: cookbook/ui_customization/how_to_customize_menu.rst [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+            labelAttributes: <attr_list>      # <span> attributes for text items without link
+            childrenAttributes: <attr_list>   # <ul> item attributes for nested lists
+            showNonAuthorized: <boolean>      # show for non-authorized users
+            display: <boolean>                # disable showing of menu item
+            displayChildren: <boolean>        # disable showing of menu item children
+
+Organize the Navigation trees
+*****************************
+
+The next step is to put together a tree for these menu items All trees are built under the ``tree`` key.
```
Comment (damien-carcel) [inline cookbook/ui_customization/how_to_customize_menu.rst]: "." after "menu items"

File: cookbook/ui_customization/how_to_customize_menu.rst (hunk 3 above)
Comment (mickaelandrieu) [inline cookbook/ui_customization/how_to_customize_menu.rst]: shame on me!

**Comment by mickaelandrieu** [comment]
@damien-carcel thanks for review, typos fixed!

**Pull request merged**